contentgrid-spring-querydsl/src/main/java/com/contentgrid/spring/querydsl/predicate/EqualsIgnoreCase.java

```diff
@@ -11,7 +11,9 @@
  * Compares strings in a case-insensitive way.
  *
  * This predicate only supports strings and can not be used with other types.
+ * @deprecated : Use {@link Text.EqualsIgnoreCase} instead.
  */
+@Deprecated(since = "v0.15.2", forRemoval = true)
 public class EqualsIgnoreCase extends AbstractSimpleQuerydslPredicateFactory<StringPath, String> {
```

Comment:
I think you can make this one extend `Text.EqualsIgnoreCase` instead? 
Keeping the class for backwards compatibility, but already delegating the implementation to the new version.